gutterball/src/main/java/org/candlepin/gutterball/model/Event.java

```diff
@@ -310,4 +306,9 @@ public void setStatus(Status status) {
         this.status = status;
     }
 
+    public void clearJsonFields() {
```

Comment:
Would it make sense to automatically call this method via on-persist/commit hook, depending on the event state?